apps/src/lab2/lab2Redux.ts

```diff
@@ -291,6 +291,7 @@ export const isReadOnlyWorkspace = (state: RootState) => {
   const isEditMode = !!getAppOptionsEditBlocks();
   const isEditingExemplar = getAppOptionsEditingExemplar();
   const isViewingExemplar = getAppOptionsViewingExemplar();
+  const isWidgetView = !!state.lab.levelProperties?.widgetView;
```

Comment:
This needs to be a `boolean` and not `undefined` for this function's return value type.